tools/devctr/Dockerfile

```diff
@@ -98,8 +99,13 @@ RUN cd /tmp/poetry \
 ENV VIRTUAL_ENV=$VENV
 ENV PATH=$VENV/bin:$PATH
 
+# apt-get installs it globally, to manually copy it into the venv
+RUN cp /usr/lib/python3/dist-packages/seccomp.cpython-312-"$ARCH"-linux-gnu.so "$VENV"/lib/python3.12/site-packages/
```

Comment:
nit: is there a reason it goes to a separate RUN?

Reply:
Just didn't feel like it belonged in the previous one